src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/ReferToJudgeReason.java

```diff
@@ -19,26 +19,44 @@
+import static uk.gov.hmcts.sptribs.caseworker.model.ReferralReason.POSTPONEMENT_REQUEST;
+import static uk.gov.hmcts.sptribs.caseworker.model.ReferralReason.REINSTATEMENT_REQUEST;
+import static uk.gov.hmcts.sptribs.caseworker.model.ReferralReason.RULE_27_REQUEST;
+import static uk.gov.hmcts.sptribs.caseworker.model.ReferralReason.SET_ASIDE_REQUEST;
+import static uk.gov.hmcts.sptribs.caseworker.model.ReferralReason.STAY_REQUEST;
+import static uk.gov.hmcts.sptribs.caseworker.model.ReferralReason.STRIKE_OUT_REQUEST;
+import static uk.gov.hmcts.sptribs.caseworker.model.ReferralReason.TIME_EXTENSION_REQUEST;
+import static uk.gov.hmcts.sptribs.caseworker.model.ReferralReason.WITHDRAWAL_REQUEST;
+import static uk.gov.hmcts.sptribs.caseworker.model.ReferralReason.WRITTEN_REASONS_REQUEST;
+import static uk.gov.hmcts.sptribs.ciccase.model.State.AwaitingHearing;
+import static uk.gov.hmcts.sptribs.ciccase.model.State.CaseClosed;
+import static uk.gov.hmcts.sptribs.ciccase.model.State.CaseManagement;
 
 public class ReferToJudgeReason implements CcdPageConfiguration {
 
     private final Map<ReferralReason, Set<State>> permittedStatesByReason =
         Map.ofEntries(
-            new SimpleEntry<>(ReferralReason.CORRECTIONS, of(State.CaseClosed)),
-            new SimpleEntry<>(ReferralReason.LISTED_CASE, of(State.AwaitingHearing)),
-            new SimpleEntry<>(ReferralReason.LISTED_CASE_WITHIN_5_DAYS, of(State.AwaitingHearing)),
-            new SimpleEntry<>(ReferralReason.LISTING_DIRECTIONS, complementOf(of(State.AwaitingHearing))),
-            new SimpleEntry<>(ReferralReason.NEW_CASE, of(State.Submitted)),
-            new SimpleEntry<>(ReferralReason.POSTPONEMENT_REQUEST, of(State.AwaitingHearing)),
-            new SimpleEntry<>(ReferralReason.REINSTATEMENT_REQUEST, of(State.CaseClosed)),
-            new SimpleEntry<>(ReferralReason.RULE_27_REQUEST, complementOf(of(State.CaseClosed))),
-            new SimpleEntry<>(ReferralReason.SET_ASIDE_REQUEST, of(State.CaseClosed)),
-            new SimpleEntry<>(ReferralReason.STAY_REQUEST, complementOf(of(State.CaseClosed))),
-            new SimpleEntry<>(ReferralReason.STRIKE_OUT_REQUEST, complementOf(of(State.CaseClosed))),
-            new SimpleEntry<>(ReferralReason.TIME_EXTENSION_REQUEST, complementOf(of(State.CaseClosed))),
-            new SimpleEntry<>(ReferralReason.WITHDRAWAL_REQUEST, complementOf(of(State.CaseClosed))),
-            new SimpleEntry<>(ReferralReason.WRITTEN_REASONS_REQUEST, of(State.CaseClosed)),
-            new SimpleEntry<>(ReferralReason.OTHER, allOf(State.class))
+            new SimpleEntry<>(CORRECTIONS, of(CaseClosed)),
+            new SimpleEntry<>(LISTED_CASE, of(AwaitingHearing)),
+            new SimpleEntry<>(LISTED_CASE_WITHIN_5_DAYS, of(AwaitingHearing)),
+            new SimpleEntry<>(LISTING_DIRECTIONS, complementOf(of(AwaitingHearing))),
+            new SimpleEntry<>(NEW_CASE, of(CaseManagement)),
```

Comment:
'New case' referral reason needs to be added to both 'Case Management' and 'Submitted' states. So, 'Submitted' state should be retained here. The respective test cases need to be added and the same changes can be completed for Refer to LO too.

Reply:
Introduced submitted again